### DISCUSSION TITLE
👋 Welcome to the new home of the Automad community

### DISCUSSION THREAD

**Discussion by marcantondahmen** [automadcms/discussions#1] [Announcements]
After many years of discussing features and exchanging knowledge about [Automad](https://automad.org) on a separate forum, it is time to move on and bring the community closer to the actual project here on GitHub. The [old forum](https://discuss.automad.org) will still be available for reference and will continue as a large knowledge base for developers as well as users.

**Comment by retry429** [comment]
Hi @marcantondahmen great idea!
Pls don't forget to update the README.MD files, under "Community" the link leads to the old community page, or simple add the new community :)

Regards,
Philipp

**Comment by marcantondahmen** [comment]
Hi, thanks for reporting this. Are you sure? The link should just be redirected. Or maybe we are referring to different READMEs?

**Comment by Imerion** [comment]
Nice! Great to see you here! Automad is a fantastic piece of software and I can't wait to try out version 2.

**Comment by hansiallen** [comment]
Hi @marcantondahmen !
I'm not sure if this was intentional or not but I think opening discussions hasn't been enabled for people (I think that this might need to be enabled manually in the settings for the GitHub account)
Overall though, I've played around with the templating system a bit though (not yet released anything, but might do in future) and it seems pretty cool! I also like the ease of use of the CMS itself!
![(What happens when you try to open a discussion)](https://github.com/automadcms/discussions/assets/49568496/ab69a4b8-26a9-42df-9ba5-57a171f9925d)
Above is a screenshot of the message that I get when attempting to open a discussion.
Sincerely,
Hansi

**Reply by marcantondahmen** [reply]
Thanks for reporting this! Much appreciated! It should work now.
